src/libpgmoneta/utils.c

```diff
@@ -826,7 +825,9 @@ pgmoneta_set_proc_title(int argc, char** argv, char* s1, char* s2)
    {
       return;
    }
+
 #ifdef HAVE_LINUX
+   char title[MAX_PROCESS_TITLE_LENGTH];
```

Comment:
Lets move this back, and do a `memset()` on it after the `config` assignment

Reply:
Oh didn't see that, right away